modules/systemSettings.js

```diff
@@ -0,0 +1,321 @@
+function SystemSettings() {
+
+  this.systemsettings = {
```

Comment:
I understand the discomfort here. It is not possible to read the value of settings as a whole. It is also not possible to read the value of a setting without knowing the name of the setting to get. Here is the get method: SettingsLock.get(settingName);  I am unsure of how to access the settings without holding dummy values, because I would not know what to provide as a 'settingName' without a dummy value.